supportedLangaugeDockerfiles/rust/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rs-base
+
+WORKDIR /app
+
+# Copy the entire application
+COPY . .
+
+# Build dependencies to cache them
+RUN cargo build --release
+
+# Make the run.sh script executable
+RUN chmod +x /app/.hxckr/run.sh
+
+# Make the run.sh script executable
```

Comment:
I guess this comment is incorrect?  Or is the command below incorrect?

Reply:
Yes it's correct, gives the container a bash entry to be able to run this `runCommand: "/app/.hxckr/run.sh"` which is in the dir all challenge users will be taking on.